extensions-builtin/sd_forge_controlnet/lib_controlnet/controlnet_ui/controlnet_ui_group.py

```diff
@@ -1172,6 +1184,8 @@ def register_callbacks(self):
         self.register_send_dimensions()
         self.register_run_annotator()
         self.register_sync_batch_dir()
+        self.register_shift_upload_mask()
```

Comment:
shift_upload_mask now takes width_slider and height_slider as input. Moving it out of core callbacks.